packages/dumb/src/components/stats/cbg-percentage/cbg-percentage-bar.hook.tsx

```diff
@@ -63,7 +63,7 @@ export const useCBGPercentageBar = (props: CBGPercentageBarHookProps): CBGPercen
       case CBGStatType.TimeInRange:
         return formatDuration(value, { condensed: true })
       case CBGStatType.ReadingsInRange:
-        return value.toString()
+        return (Math.round(value * 10) / 10).toString()
```

Comment:
I've seen this computation `Math.round(value * 10) / 10` a lot of time in your PRs in dumb, it might be worth creating a method to compute it?

Reply:
Any suggestion on where to add this method?

Reply:
`format.util.ts` would seem appropriate

Reply:
As discussed, not necessary for now